data/save.json

```diff
@@ -1,5 +1,62 @@
 {"version":2,"weight":0,"name":"save","elements":[
-{"id":"permakey","name":"permakey","type":"number"},
-{"id":"sigil","name":"sigil","type":"number"},
-{"id":"safetysave","name":"","type":"number"}
+{"id":"TutorialSaddle_Save","name":"TutorialSaddle_Save","type":"number"},
```

Comment:
I don't know if I like the _Save attached to the end of each name. It makes it rather cumbersome with the box as well. IDK, we can change it later if we want

Reply:
In the context of the new v4 guide wording, I've removed the text that always says "Make sure to write down blah blah blah" to make it less bulky, so I figured having the _Save will help remind new users that those are saves since they appear in-line with instructions with little context otherwise.